core/helpers.py

```diff
@@ -11,6 +11,13 @@
 )
 
 
+def instantiate_subclass(base_class, subclass_name):
+    subclasses = {}
```

Comment:
FTR, you can change this to `subclasses = {cls.__name__: cls for cls in base_class.__subclasses__()}`